[2018-10-30 16:57:09] ===检测备份目录
[2018-10-30 16:57:09] 备份目录''不存在
[2018-10-30 16:57:09] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 16:57:09] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 16:57:09] ====================开始sleep,进入下一次循环
[2018-10-30 16:57:59] ===检测备份目录
[2018-10-30 16:57:59] 备份目录''不存在
[2018-10-30 16:57:59] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 16:57:59] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 16:57:59] ====================开始sleep,进入下一次循环
[2018-10-30 16:58:37] ===检测备份目录
[2018-10-30 16:58:37] 备份目录''不存在
[2018-10-30 16:58:37] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 16:58:37] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 16:58:37] ====================开始sleep,进入下一次循环
[2018-10-30 16:59:05] ===检测备份目录
[2018-10-30 16:59:05] 备份目录'/data/code/github/repository/mysql_repo/mysql_backup/log_dir/shell.log'不存在
[2018-10-30 16:59:05] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 16:59:05] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 16:59:05] ====================开始sleep,进入下一次循环
[2018-10-30 16:59:34] ===检测备份目录
[2018-10-30 16:59:34] 备份目录'/data/code/github/repository/mysql_repo/mysql_backup/log_dir/shell.log'不存在
[2018-10-30 16:59:34] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 16:59:34] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 16:59:34] ====================开始sleep,进入下一次循环
[2018-10-30 16:59:48] ===检测备份目录
[2018-10-30 16:59:48] 备份目录'/data/code/github/repository/mysql_repo/mysql_backup/log_dir/shell.log'不存在
[2018-10-30 16:59:48] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 16:59:48] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 16:59:48] ====================开始sleep,进入下一次循环
[2018-10-30 17:00:48] ===检测备份目录
[2018-10-30 17:00:48] 备份目录'/data/code/github/repository/mysql_repo/mysql_backup/log_dir/shell.log'不存在
[2018-10-30 17:00:48] [TestDB_20181030 TestDB 172.16.112.10:10000] 尝试备份Fail
[2018-10-30 17:00:48] ====================进入检测逻辑,可以ctrl+c
[2018-10-30 17:00:48] ====================开始sleep,进入下一次循环
[2018-10-31 11:37:09] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P10001 -urepl_user -prepl_user  --read-from-remote-server --raw --stop-never --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_10001/REPORTED/
[2018-10-31 11:37:09] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:37:09] [TestDB-172.16.112.10:10001] 进入binlog上报逻辑
[2018-10-31 11:37:09] ======开始sleep,进入下一次循环
[2018-10-31 11:37:28] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user  --read-from-remote-server --raw --stop-never --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:37:28] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:37:28] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:37:28] ======开始sleep,进入下一次循环
[2018-10-31 11:43:12] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:12] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:12] ======开始sleep,进入下一次循环
[2018-10-31 11:43:13] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:13] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:13] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:13] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:13] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:13] dir_size:1628,slave_size:3124
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:13] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:13] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:13] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:13] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:13] ======开始sleep,进入下一次循环
[2018-10-31 11:43:14] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:14] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:14] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:14] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:14] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:14] dir_size:4536,slave_size:6266
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:14] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:14] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:14] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:14] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:14] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:14] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:16] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:16] ======开始sleep,进入下一次循环
[2018-10-31 11:43:17] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:17] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:17] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:17] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:17] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:17] dir_size:7699,slave_size:10725
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:17] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:17] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:17] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:17] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:17] ======开始sleep,进入下一次循环
[2018-10-31 11:43:18] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:18] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:18] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:18] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:19] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:19] dir_size:12159,slave_size:14180
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:19] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:19] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:19] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:19] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:19] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:19] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:21] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:21] ======开始sleep,进入下一次循环
[2018-10-31 11:43:22] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:22] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:22] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:22] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:22] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:22] dir_size:15613,slave_size:18639
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:22] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:22] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:22] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:22] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:22] ======开始sleep,进入下一次循环
[2018-10-31 11:43:23] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:23] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:23] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:23] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:23] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:23] dir_size:20073,slave_size:21803
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:23] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:23] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:23] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:23] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:23] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:23] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:25] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:25] ======开始sleep,进入下一次循环
[2018-10-31 11:43:26] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:26] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:26] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:26] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:26] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:26] dir_size:23236,slave_size:26262
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:26] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:26] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:26] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:26] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:26] ======开始sleep,进入下一次循环
[2018-10-31 11:43:27] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:27] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:27] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:27] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:28] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:28] dir_size:27696,slave_size:29426
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:28] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:28] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:28] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:28] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:28] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:28] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:30] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:30] ======开始sleep,进入下一次循环
[2018-10-31 11:43:31] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:31] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:31] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:31] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:31] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:31] dir_size:30859,slave_size:33885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:31] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:31] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:31] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:31] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:31] ======开始sleep,进入下一次循环
[2018-10-31 11:43:32] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:32] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:32] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:32] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:32] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:32] dir_size:35319,slave_size:37340
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:32] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:32] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:32] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:32] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:32] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:32] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:34] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:34] ======开始sleep,进入下一次循环
[2018-10-31 11:43:35] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:35] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:35] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:35] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:35] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:35] dir_size:38773,slave_size:41799
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:35] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:35] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:35] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:35] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:35] ======开始sleep,进入下一次循环
[2018-10-31 11:43:36] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:36] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:36] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:36] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:36] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:37] dir_size:43233,slave_size:44963
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:37] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:37] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:37] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:37] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:37] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:37] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:39] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:39] ======开始sleep,进入下一次循环
[2018-10-31 11:43:40] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:40] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:40] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:40] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:40] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:40] dir_size:46396,slave_size:49422
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:40] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:40] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:40] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:40] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:40] ======开始sleep,进入下一次循环
[2018-10-31 11:43:41] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:41] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:41] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:41] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:41] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:41] dir_size:50856,slave_size:52586
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:41] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:41] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:41] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:41] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:41] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:41] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:43] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:43] ======开始sleep,进入下一次循环
[2018-10-31 11:43:44] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:44] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:44] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:44] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:44] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:44] dir_size:54019,slave_size:57045
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:44] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:44] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:44] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:44] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:44] ======开始sleep,进入下一次循环
[2018-10-31 11:43:45] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:45] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:45] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:45] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:45] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:45] dir_size:58479,slave_size:60209
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:45] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:45] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:45] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:45] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:45] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:45] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:47] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:47] ======开始sleep,进入下一次循环
[2018-10-31 11:43:48] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:48] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:48] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:48] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:48] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:49] dir_size:61933,slave_size:64959
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:49] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:49] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:49] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:49] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:49] ======开始sleep,进入下一次循环
[2018-10-31 11:43:50] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:50] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:50] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:50] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:50] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:50] dir_size:66393,slave_size:68123
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:50] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:50] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:50] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:50] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:50] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:50] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:52] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:52] ======开始sleep,进入下一次循环
[2018-10-31 11:43:53] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:53] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:53] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:53] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:53] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:53] dir_size:69556,slave_size:72582
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:53] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:53] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:53] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:53] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:53] ======开始sleep,进入下一次循环
[2018-10-31 11:43:54] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:54] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:54] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:54] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:54] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:54] dir_size:74016,slave_size:75746
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:54] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:54] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:54] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:54] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:54] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:54] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:56] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:43:56] ======开始sleep,进入下一次循环
[2018-10-31 11:43:57] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:57] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:57] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:57] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:57] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:57] dir_size:77179,slave_size:80205
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:57] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:43:57] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:57] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:57] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:57] ======开始sleep,进入下一次循环
[2018-10-31 11:43:59] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:43:59] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:43:59] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:43:59] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:43:59] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:43:59] dir_size:81639,slave_size:83369
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:43:59] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:43:59] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:43:59] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:43:59] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:43:59] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:43:59] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:01] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:01] ======开始sleep,进入下一次循环
[2018-10-31 11:44:02] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:02] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:02] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:02] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:02] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:02] dir_size:84802,slave_size:87828
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:02] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:02] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:02] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:02] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:02] ======开始sleep,进入下一次循环
[2018-10-31 11:44:03] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:03] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:03] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:03] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:03] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:03] dir_size:89262,slave_size:91283
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:03] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:03] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:03] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:03] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:03] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:03] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:05] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:05] ======开始sleep,进入下一次循环
[2018-10-31 11:44:06] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:06] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:06] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:06] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:06] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:06] dir_size:92716,slave_size:95742
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:06] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:06] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:06] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:06] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:06] ======开始sleep,进入下一次循环
[2018-10-31 11:44:07] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:07] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:07] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:07] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:07] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:08] dir_size:97176,slave_size:98906
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:08] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:08] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:08] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:08] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:08] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:08] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:10] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:10] ======开始sleep,进入下一次循环
[2018-10-31 11:44:11] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:11] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:11] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:11] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:11] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:11] dir_size:100339,slave_size:103365
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:11] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:11] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:11] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:11] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:11] ======开始sleep,进入下一次循环
[2018-10-31 11:44:12] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:12] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:12] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:12] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:12] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:12] dir_size:104799,slave_size:106529
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:12] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:12] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:12] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:12] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:12] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:12] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:14] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:14] ======开始sleep,进入下一次循环
[2018-10-31 11:44:15] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:15] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:15] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:15] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:15] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:15] dir_size:107962,slave_size:110988
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:15] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:15] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:15] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:15] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:15] ======开始sleep,进入下一次循环
[2018-10-31 11:44:16] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:16] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:16] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:16] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:16] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:16] dir_size:112422,slave_size:114152
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:16] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:16] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:16] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:16] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:17] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:17] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:18] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:18] ======开始sleep,进入下一次循环
[2018-10-31 11:44:19] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:19] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:19] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:19] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:20] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:20] dir_size:115876,slave_size:118902
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:20] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:20] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:20] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:20] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:20] ======开始sleep,进入下一次循环
[2018-10-31 11:44:21] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:21] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:21] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:21] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:21] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:21] dir_size:120336,slave_size:122066
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:21] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:21] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:21] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:21] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:21] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:21] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:23] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:23] ======开始sleep,进入下一次循环
[2018-10-31 11:44:24] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:24] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:24] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:24] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:24] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:24] dir_size:123499,slave_size:126525
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:24] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:24] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:24] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:24] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:24] ======开始sleep,进入下一次循环
[2018-10-31 11:44:25] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:25] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:25] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:25] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:25] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:25] dir_size:127959,slave_size:129689
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:25] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:25] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:25] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:25] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:25] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:25] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:27] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:27] ======开始sleep,进入下一次循环
[2018-10-31 11:44:28] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:28] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:28] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:28] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:28] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:28] dir_size:131122,slave_size:134148
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:28] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:28] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:28] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:28] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:28] ======开始sleep,进入下一次循环
[2018-10-31 11:44:30] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:30] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:30] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:30] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:30] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:30] dir_size:135582,slave_size:137312
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:30] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:30] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:30] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:30] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:30] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:30] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:32] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:32] ======开始sleep,进入下一次循环
[2018-10-31 11:44:33] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:33] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:33] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:33] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:33] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:33] dir_size:138745,slave_size:141771
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:33] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:33] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:33] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:33] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:33] ======开始sleep,进入下一次循环
[2018-10-31 11:44:34] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:34] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:34] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:34] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:34] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:34] dir_size:143205,slave_size:145226
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:34] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:34] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:34] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:34] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:34] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:34] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:36] 单独拉取binlog成功:binlog.000001
[2018-10-31 11:44:36] ======开始sleep,进入下一次循环
[2018-10-31 11:44:37] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:37] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:37] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:37] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:37] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:37] dir_size:146659,slave_size:149685
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:37] dir_size:122632885,slave_size:122632885
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’
mv: cannot move ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000001’ to ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/’: Not a directory
[2018-10-31 11:44:37] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:37] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:37] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:37] ======开始sleep,进入下一次循环
[2018-10-31 11:44:38] ======进入mysqlbinlog监控逻辑
[2018-10-31 11:44:38] [TestDB-172.16.112.10:11000] 进入mysqlbinlog监控逻辑
[2018-10-31 11:44:38] [TestDB 172.16.112.10:11000] mysqlbinlog进程正常
[2018-10-31 11:44:38] ======进入binlog上报逻辑,不能使用ctrl+c
[2018-10-31 11:44:38] [TestDB-172.16.112.10:11000] 进入binlog上报逻辑
[2018-10-31 11:44:38] dir_size:151119,slave_size:152849
‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/binlog.000002’ -> ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/FAIL/binlog.000002’
[2018-10-31 11:44:38] dir_size:,slave_size:122632885
mv: cannot stat ‘/data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/SUCC/20181026/binlog.000001’: No such file or directory
[2018-10-31 11:44:38] 开始单独拉失败的binlog:binlog.000002
[2018-10-31 11:44:38] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 11:44:38] 单独拉取binlog成功:binlog.000002
[2018-10-31 11:44:39] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 11:44:39] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 14:08:59] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 14:09:01] 单独拉取binlog成功:binlog.000001
[2018-10-31 14:09:01] ======开始sleep,进入下一次循环
[2018-10-31 14:10:09] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 14:10:09] 单独拉取binlog成功:binlog.000002
[2018-10-31 14:10:09] ======开始sleep,进入下一次循环
[2018-10-31 14:10:25] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
[2018-10-31 14:10:27] 单独拉取binlog成功:binlog.000001
[2018-10-31 14:10:27] ======开始sleep,进入下一次循环
[2018-10-31 14:11:54] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
mysqlbinlog: [Warning] Using a password on the command line interface can be insecure.
[2018-10-31 14:12:14] 单独拉取binlog成功:binlog.000002
[2018-10-31 14:12:14] ======开始sleep,进入下一次循环
[2018-10-31 14:14:25] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
mysqlbinlog: [Warning] Using a password on the command line interface can be insecure.
[2018-10-31 14:14:47] 单独拉取binlog成功:binlog.000001
[2018-10-31 14:14:47] ======开始sleep,进入下一次循环
[2018-10-31 14:18:11] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000002 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
mysqlbinlog: [Warning] Using a password on the command line interface can be insecure.
[2018-10-31 14:18:21] 单独拉取binlog成功:binlog.000002
[2018-10-31 14:18:21] 开始单独拉失败的binlog:binlog.000001
[2018-10-31 14:18:21] /data/code/github/repository/mysql_repo/mysql_backup/common/mysqlbinlog -h172.16.112.10 -P11000 -urepl_user -prepl_user binlog.000001 --read-from-remote-server --raw --result-file /data/MySQL_BACKUP/BINARYLOG_BACKUP/TESTDB/172.16.112.10_11000/REPORTED/
mysqlbinlog: [Warning] Using a password on the command line interface can be insecure.
[2018-10-31 14:18:33] 单独拉取binlog成功:binlog.000001
[2018-10-31 14:18:33] ======开始sleep,进入下一次循环
